openpype/hosts/houdini/plugins/create/create_usd.py

```diff
@@ -13,7 +15,6 @@ class CreateUSD(plugin.HoudiniCreator):
     enabled = False
 
     def create(self, subset_name, instance_data, pre_create_data):
-        import hou  # noqa
 
         instance_data.pop("active", None)
         instance_data.update({"node_type": "usd"})
```

Comment:
i know it's not related to this PR. is the node_type "usd" from external plugin or not? Fir some reasons it's not working ang gives the error for both creation from TAB menu and that from the publisher.
![image](https://user-images.githubusercontent.com/64118225/233537760-6314bafc-cdfc-45f4-b665-8f533717a0e9.png)

Reply:
Was able to reproduce here. The stack trace was:

```python
Traceback (most recent call last):
  File "S:\openpype\OpenPype\openpype\pipeline\create\context.py", line 1996, in _create_with_unified_error
    result = creator.create(*args, **kwargs)
  File "S:\openpype\OpenPype\openpype\hosts\houdini\plugins\create\create_usdrender.py", line 26, in create
    pre_create_data)  # type: CreatedInstance
  File "S:\openpype\OpenPype\openpype\hosts\houdini\api\plugin.py", line 191, in create
    self)
  File "S:\openpype\OpenPype\openpype\pipeline\create\context.py", line 914, in __init__
    data = copy.deepcopy(data or {})
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 241, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 180, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 281, in _reconstruct
    state = deepcopy(state, memo)
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 241, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "C:\PROGRA~1\SIDEEF~1\HOUDIN~1.435\python37\lib\copy.py", line 169, in deepcopy
    rv = reductor(4)
TypeError: can't pickle SwigPyObject objects
```

The issue is with this line:
```python
        instance_data["parent"] = hou.node("/stage")
```

`hou.Node` instances can't be deep copied and thus can't be added to the instance data for the `create` method.
It'd need to be set as transient data afterwards.

This should definitely be a separate issue and PR. Will take a quick look.

Reply:
Created https://github.com/ynput/OpenPype/issues/4889 to track the issue.

Reply:
I see, sorry I should have checked this before responding to https://github.com/ynput/OpenPype/issues/4889. Couldn't we store the `/stage` string on `"parent"` instead, why do we need the `hou.Node` object?

Reply:
> Couldn't we store the /stage string on "parent" instead, why do we need the hou.Node object?

In this case 100% we could, yes.